distributor/cmd/main.go

```diff
@@ -98,16 +107,38 @@ func _main(args []string, env envConfig) int {
 	default:
 		createNATSClientConnection()
 	}
-
+*/
 	return 0
 }
 
+  func connectionHTTP(connectionType string,  wg *sync.WaitGroup) {
```

Comment:
This function kind of mixes the initialisation of the HTTP event polling client, and the HTTP Server that is used to forward events to the Keptn API/NATS server - these two different responsibilities should be split up in their own go routines